### PR TITLE
Fix #1576: Excessive logging of CreatePersonalizedOfflineSignaturePayloadResponse

### LINKED ISSUE #1576: Excessive logging of CreatePersonalizedOfflineSignaturePayloadResponse
This method is loggin too much detail and PII

```app-1  | 2024-06-24T15:39:06.917Z  INFO 156 --- [powerauth-java-server] [nio-8080-exec-3] [8685d97eafa5483586de3bc4c6e2544c-d83cf2dfb68941a8] i.g.s.p.a.s.c.api.SignatureController    : CreatePersonalizedOfflineSignaturePayloadRequest succeeded: ObjectResponse(super=Response(status=OK), responseObject=CreatePersonalizedOfflineSignaturePayloadResponse(offlineData=16f97a47-22f3-427a-b004-cba89fc848cf
app-1  | Approve Payment
app-1  | Please approve the payment of 100.00 CZK to account CZ2730300000001165254011.
app-1  | A1*A100.00CZK*ICZ2730300000001165254011
app-1  | B
app-1  | cCCv/DSs1Xx4birdoo2wLQ==
app-1  | 1MEUCIQCMw0Yq6LocKCJUFBiSv1YbgmQ45MEEurNHyIhjOo1ynQIgWLVIfNL4s+vXI9Fg4vAVh8RG+78La6sM9pLESn7BOBE=))
```

### PR DESCRIPTION
**Pull request by banterCZ**

### CODE REVIEW

File: powerauth-java-server/src/main/java/io/getlime/security/powerauth/app/server/controller/api/SignatureController.java [hunk 1]
```diff
@@ -83,7 +83,8 @@ public ObjectResponse<VerifySignatureResponse> verifySignature(@RequestBody Obje
     public ObjectResponse<CreatePersonalizedOfflineSignaturePayloadResponse> createPersonalizedOfflineSignaturePayload(@RequestBody ObjectRequest<CreatePersonalizedOfflineSignaturePayloadRequest> request) throws Exception {
         logger.info("CreatePersonalizedOfflineSignaturePayloadRequest received: {}", request);
```
Comment (jnpsk) [inline powerauth-java-server/src/main/java/io/getlime/security/powerauth/app/server/controller/api/SignatureController.java:84]: Doesn't logging of the request object cause same issue?
Maybe we could log only the activation ID on info level.

Comment (zcgandcomp) [reply powerauth-java-server/src/main/java/io/getlime/security/powerauth/app/server/controller/api/SignatureController.java:84]: yes, do not log the whole object on info level. And if so apply our logging guideline. action: {}, state: {},...

**Review by zcgandcomp — CHANGES_REQUESTED** [review]

**Review by romanstrobl — APPROVED** [review]
Nice cleanup!

File: powerauth-java-server/src/main/java/io/getlime/security/powerauth/app/server/controller/api/SignatureController.java [hunk 2]
```diff
@@ -81,9 +81,11 @@ public ObjectResponse<VerifySignatureResponse> verifySignature(@RequestBody Obje
      */
     @PostMapping("/offline/personalized/create")
     public ObjectResponse<CreatePersonalizedOfflineSignaturePayloadResponse> createPersonalizedOfflineSignaturePayload(@RequestBody ObjectRequest<CreatePersonalizedOfflineSignaturePayloadRequest> request) throws Exception {
-        logger.info("CreatePersonalizedOfflineSignaturePayloadRequest received: {}", request);
+        logger.info("CreatePersonalizedOfflineSignaturePayloadRequest received.");
+        logger.debug("CreatePersonalizedOfflineSignaturePayloadRequest received: {}", request);
         final ObjectResponse<CreatePersonalizedOfflineSignaturePayloadResponse> response = new ObjectResponse<>(offlineSignatureService.createPersonalizedOfflineSignaturePayload(request.getRequestObject()));
-        logger.info("CreatePersonalizedOfflineSignaturePayloadRequest succeeded: {}", response);
+        logger.info("CreatePersonalizedOfflineSignaturePayloadRequest succeeded.");
```
Comment (zcgandcomp) [inline powerauth-java-server/src/main/java/io/getlime/security/powerauth/app/server/controller/api/SignatureController.java:87]:
```suggestion
        logger.info("action: createPersonalizedOfflineSignaturePayload, state: succeeded");
```

File: powerauth-java-server/src/main/java/io/getlime/security/powerauth/app/server/controller/api/SignatureController.java [hunk 3]
```diff
@@ -81,9 +81,11 @@ public ObjectResponse<VerifySignatureResponse> verifySignature(@RequestBody Obje
      */
     @PostMapping("/offline/personalized/create")
     public ObjectResponse<CreatePersonalizedOfflineSignaturePayloadResponse> createPersonalizedOfflineSignaturePayload(@RequestBody ObjectRequest<CreatePersonalizedOfflineSignaturePayloadRequest> request) throws Exception {
-        logger.info("CreatePersonalizedOfflineSignaturePayloadRequest received: {}", request);
+        logger.info("CreatePersonalizedOfflineSignaturePayloadRequest received.");
```
Comment (zcgandcomp) [inline powerauth-java-server/src/main/java/io/getlime/security/powerauth/app/server/controller/api/SignatureController.java:84]:
```suggestion
        logger.info("action: createPersonalizedOfflineSignaturePayload, state: initiated, activationId: {}", request.getActivationod());
```

**Review by zcgandcomp — CHANGES_REQUESTED** [review]

File: powerauth-java-server/src/main/java/io/getlime/security/powerauth/app/server/controller/api/SignatureController.java [hunk 4]
```diff
@@ -96,9 +98,11 @@ public ObjectResponse<CreatePersonalizedOfflineSignaturePayloadResponse> createP
      */
     @PostMapping("/offline/non-personalized/create")
     public ObjectResponse<CreateNonPersonalizedOfflineSignaturePayloadResponse> createNonPersonalizedOfflineSignaturePayload(@RequestBody ObjectRequest<CreateNonPersonalizedOfflineSignaturePayloadRequest> request) throws Exception {
-        logger.info("CreateNonPersonalizedOfflineSignaturePayloadRequest received: {}", request);
+        logger.info("CreateNonPersonalizedOfflineSignaturePayloadRequest received.");
```
Comment (zcgandcomp) [inline powerauth-java-server/src/main/java/io/getlime/security/powerauth/app/server/controller/api/SignatureController.java:101]: the same refactor as above
use action: CreateNonPersonalizedOfflineSignaturePayloadRequest, state: received and identifier relevant to the request

File: powerauth-java-server/src/main/java/io/getlime/security/powerauth/app/server/controller/api/SignatureController.java [hunk 5]
```diff
@@ -96,9 +98,11 @@ public ObjectResponse<CreatePersonalizedOfflineSignaturePayloadResponse> createP
...
+        logger.debug("CreateNonPersonalizedOfflineSignaturePayloadRequest received: {}", request);
         final ObjectResponse<CreateNonPersonalizedOfflineSignaturePayloadResponse> response = new ObjectResponse<>(offlineSignatureService.createNonPersonalizedOfflineSignaturePayload(request.getRequestObject()));
-        logger.info("CreateNonPersonalizedOfflineSignaturePayloadRequest succeeded: {}", response);
+        logger.info("CreateNonPersonalizedOfflineSignaturePayloadRequest succeeded.");
```
Comment (zcgandcomp) [inline powerauth-java-server/src/main/java/io/getlime/security/powerauth/app/server/controller/api/SignatureController.java:104]: here add action: createNonPersonalizedOfflineSignaturePayload and state: succeeded.

**Review by zcgandcomp — APPROVED** [review]
Looks OK

**Pull request merged**